pkg/log/log.go

```diff
@@ -40,7 +43,23 @@ func NewLogger(level string, output string) Logger {
 		log = zerolog.New(file)
 	}
 
-	return Logger{Logger: log.With().Caller().Timestamp().Logger()}
+	var auditLog zerolog.Logger
+
+	if audit != "" {
+		auditFile, auditErr := os.OpenFile(audit, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0600)
+		if auditErr != nil {
+			panic(auditErr)
+		}
+
+		auditLog = zerolog.New(auditFile)
+
+		return Logger{Logger: log.With().Caller().Timestamp().Logger()},
+			Logger{Logger: auditLog.With().Timestamp().Logger()}
+	}
+
+	// if audit file does not exist, redirect the output to /dev/null
+	return Logger{Logger: log.With().Caller().Timestamp().Logger()},
+		Logger{Logger: zerolog.New(ioutil.Discard)}
```

Comment:
@rchincha, if the "audit" field is missing in the config, the output is redirected to **/dev/null**